alfresco-transform-tika/alfresco-transform-tika-boot/src/test/resources/quick.odf_metadata.json

```diff
@@ -1,6 +1,6 @@
-{
-  "{http://www.alfresco.org/model/content/1.0}description" : "Gym class featuring a brown fox and lazy dog",
-  "{http://www.alfresco.org/model/content/1.0}created" : 1138362922000,
-  "{http://www.alfresco.org/model/content/1.0}author" : null,
-  "{http://www.alfresco.org/model/content/1.0}title" : "The quick brown fox jumps over the lazy dog"
+{
+  "{http://www.alfresco.org/model/content/1.0}description" : "Gym class featuring a brown fox and lazy dog",
+  "{http://www.alfresco.org/model/content/1.0}created" : 1138370122000,

```

Comment:
Please revert this file. odf is not handled by IPTC extraction so should not change.